modules/persistent-storage-csi-efs-metrics-overview.adoc

```diff
@@ -0,0 +1,16 @@
+// Module included in the following assemblies:
+//
+// * storage/persistent_storage/persistent-storage-csi-aws-efs.adoc
+// 
+
+[id="efs-metrics-overview_{context}"]
+= Usage metrics overview
+
+Amazon Web Services (AWS) Elastic File Service (EFS) usage metrics allows you to monitor how much EFS space is used by {product-title}.
```

Comment:
How much space is used by EFS volumes.
https://github.com/kubernetes-sigs/aws-efs-csi-driver?tab=readme-ov-file#container-arguments-for-efs-plugin-of-efs-csi-node-daemonset

@chao007 did you check how it looks with EFS static provisioning? See below, thanks
https://github.com/kubernetes-sigs/aws-efs-csi-driver/issues/691#issuecomment-1680406531

Reply:
cc @bertinatto

Reply:
Seems missed this. Will check it today.

Reply:
```
{
  "driverConfig": {
    "aws": {
      "efsVolumeMetrics": {
        "recursiveWalk": {
          "fsRateLimit": 10,
          "refreshPeriodMinutes": 8
        },
        "state": "RecursiveWalk"
      }
    },
    "driverType": "AWS"
  },

 oc get daemonset.apps/aws-efs-csi-driver-node -o json | jq .spec.template.spec.containers[0].args
[
  "--endpoint=$(CSI_ENDPOINT)",
  "--logtostderr",
  "--v=8",
  "--vol-metrics-opt-in=true",
  "--vol-metrics-refresh-period=8",
  "--vol-metrics-fs-rate-limit=10"
]
```

Reply:
@gcharot daemonset.apps/aws-efs-csi-driver-node will update when we edit the values fsRateLimit/refreshPeriodMinutes in clustercsidriver

Reply:
@chao007 Does this mean that if you are using static provisioning that you need to set those two fields as shown above from their default values?

Reply:
@lpettyjo No need to set those fields. We can use the default one.

Reply:
@chao007 So, we simply were checking to see that static as well as dynamic provisoining is supported, since there was some concern on the static side.

Reply:
Got it.